packages/repository-modernizer/src/restructure-pom.js

```diff
@@ -128,14 +128,10 @@ var RestructurePoms = {
                 );
             }
             sdkVersion = await fetchSDKMetadata();
-            let sdkDependency = constants.SDK_DEPENDENCY_TEMPLATE.replace(
-                "${version}",
-                sdkVersion
-            );
-            uiAppsDependencyList.push(sdkDependency);
+            uiAppsDependencyList.push(constants.SDK_DEPENDENCY_TEMPLATE);
             addSdkDependencytoCoreBundles(
                 project,
-                sdkDependency,
+                constants.SDK_DEPENDENCY_TEMPLATE,
```

Comment:
Minor issue : Since it is the constant template (i.e. not edited), should we send it any longer? Maybe it can be used in the required place directly, instead of being sent as a function arg.

Reply:
Thanks, @ManasMaji, it has been addressed.